projects/igniteui-angular/src/lib/core/grid-selection.ts

```diff
@@ -688,7 +688,9 @@ export class IgxGridSelectionService {
     }
 
     private isFilteringApplied(): boolean {
-        return this.grid.filteringExpressionsTree.filteringOperands.length > 0;
+        const grid = this.grid as IgxGridBaseComponent;
+        return grid.filteringExpressionsTree.filteringOperands.length > 0 ||
+        grid.advancedFilteringExpressionsTree && grid.advancedFilteringExpressionsTree.filteringOperands.length > 0;
```

Comment:
You could use the static helper method `FilteringExpressionsTree.empty()`.

Reply:
Done